src/constants/links.ts

```diff
@@ -1 +1,2 @@
 export const SUGGEST_FEATURE_LINK = 'https://grill.hellonext.co'
+export const SUGGEST_NEW_CHAT_ROOM_LINK = 'https://forms.gle/Lo5F4U4rq3rVoth1A'
```

Comment:
https://docs.google.com/forms/d/e/1FAIpQLSdhp3ZGAH3Gxbm6xtRcd8QgjI3M2RVdUY9UqepseXKr3DXirw/viewform?entry.1674564644=Polkadot

Need to use predefined text for Hub name field

You can use this url, and just change name in the end of the link